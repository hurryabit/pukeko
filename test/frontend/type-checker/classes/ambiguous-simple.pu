class C a where
  c : Int
f : Int
f = c
