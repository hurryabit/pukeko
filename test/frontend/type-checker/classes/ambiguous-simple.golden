test/frontend/type-checker/classes/ambiguous-simple.pu:4:1: ambiguous type variable in constraint C _1
